db/auto_roll_logger_test.cc

```diff
@@ -111,45 +118,43 @@ void AutoRollLoggerTest::RollLogFileBySizeTest(AutoRollLogger* logger,
   ASSERT_TRUE(message_size == logger->GetLogFileSize());
 }
 
-uint64_t AutoRollLoggerTest::RollLogFileByTimeTest(
-    AutoRollLogger* logger, size_t time, const std::string& log_message) {
-  uint64_t expected_create_time;
-  uint64_t actual_create_time;
+void AutoRollLoggerTest::RollLogFileByTimeTest(Env* _env,
```

Comment:
Oh ok then I guess this one's name is good. Maybe the other `env` can be renamed to `default_env` (or removed entirely) since it's always DefaultEnv().